openquake/hazardlib/gsim/abrahamson_2015.py

```diff
@@ -82,11 +82,8 @@ def get_stress_factor(imt, slab):
     Returns the stress adjustment factor for the BC Hydro GMPE according to
     Abrahamson et al. (2018)
     """
-    if slab:
-        sigma_mu = BCHYDRO_SIGMA_MU[imt]["SIGMA_MU_SSLAB"]
-    else:
-        sigma_mu = BCHYDRO_SIGMA_MU[imt]["SIGMA_MU_SINTER"]
-    return sigma_mu / 1.65
+    C = BCHYDRO_SIGMA_MU[imt]
+    return (C["SIGMA_MU_SSLAB"] if slab else C["SIGMA_MU_SINTER"]) / 1.65
```

Comment:
I do not think this is equivalent. Sigma mu in the original is always divided by 1.65